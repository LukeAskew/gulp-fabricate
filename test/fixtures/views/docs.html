---
title: Document Name
---

{{#each docs}}

	<h1>{{this.name}}</h1>

	{{{content}}}

{{/each}}
