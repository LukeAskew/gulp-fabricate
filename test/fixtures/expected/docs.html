<!doctype html>
<html lang="en">
<head>

	<meta charset="utf-8">
	<meta http-equiv="x-ua-compatible" content="ie=edge">

	<title>Document Name</title>

	<meta name="viewport" content="width=device-width, initial-scale=1">

	<link rel="stylesheet" href="assets/styles/main.css">

</head>
<body>

	<h1>Javascript</h1>

	<h2>This is a markdown document.</h2>

	<p><code>var foo = 'bar';</code></p>

	<h1>Markup</h1>

	<h2>Markup</h2>

	<script src="assets/scripts/main.js"></script>

</body>
</html>
